libindy/tests/crypto.rs

```diff
@@ -195,7 +195,7 @@ mod high_cases {
         use super::*;
 
         #[test]
-        fn indy_crypto_sign_works() {
+        fn ursa_sign_works() {
```

Comment:
Seems like tests names in this file was auto-replaced from composite prefixes `indy` `crypto` to `ursa`.
Please revert these changes.